qml/Panopticon/CommentOverlay.qml

```diff
@@ -152,7 +152,7 @@ MouseArea {
 				Layout.row: 2
 				Layout.alignment: Qt.AlignRight
 
-				text: "Comment (Shift+Return)"
+				text: "Comment (Ctrl/⌘+Return)"
```

Comment:
I would like this text to be changed depending on the operating system. Any suggestions?